brion/synapseSummary.cpp

```diff
@@ -55,8 +63,8 @@ class SynapseSummary
 
         try
         {
-            SilenceHDF5 silence;
-            const std::string& datasetName = _file.getObjnameByIdx(0);
+            // SilenceHDF5 silence;
```

Comment:
rm

Reply:
Shouldn't be commented. It was for debugging.